src/aws_encryption_sdk/materials_managers/__init__.py

```diff
@@ -92,16 +152,64 @@ class DecryptionMaterialsRequest(object):
     encryption_context = attr.ib(validator=attr.validators.instance_of(dict))
 
 
-@attr.s(hash=False)
-class DecryptionMaterials(object):
+_DEFAULT_SENTINEL = object()
+
+
+@attr.s(hash=False, init=False)
+class DecryptionMaterials(CryptographicMaterials):
     """Decryption materials returned by a crypto material manager's `decrypt_materials` method.
 
     .. versionadded:: 1.3.0
 
-    :param data_key: Plaintext data key to use with message decryption
-    :type data_key: aws_encryption_sdk.structures.DataKey
-    :param bytes verification_key: Raw signature verification key
+    .. versionadded:: 1.5.0
+
+        The **algorithm**, **data_encryption_key**, **encrypted_data_keys**,
+        **encryption_context**, and **keyring_trace** parameters.
+
+    .. versionadded:: 1.5.0
+
+        All parameters are now optional.
+
+    :param Algorithm algorithm: Algorithm to use for encrypting message (optional)
+    :param DataKey data_encryption_key: Plaintext data key to use for encrypting message (optional)
+    :param encrypted_data_keys: List of encrypted data keys (optional)
+    :type encrypted_data_keys: list of :class:`EncryptedDataKey`
+    :param dict encryption_context: Encryption context tied to `encrypted_data_keys` (optional)
+    :param bytes verification_key: Raw signature verification key (optional)
+    :param keyring_trace: Any KeyRing trace entries (optional)
+    :type keyring_trace: list of :class:`KeyringTrace`
     """
 
-    data_key = attr.ib(validator=attr.validators.instance_of(DataKey))
     verification_key = attr.ib(default=None, validator=attr.validators.optional(attr.validators.instance_of(bytes)))
+
+    def __init__(self, data_key=_DEFAULT_SENTINEL, verification_key=None, **kwargs):
+        if any(
+            (
+                data_key is _DEFAULT_SENTINEL and "data_encryption_key" not in kwargs,
```

Comment:
Is this where we exclude the EDK and the data key?

Reply:
No, this is to maintain backwards compatibility with the old `DecryptionMaterials` API.